package.json

```diff
@@ -88,17 +92,17 @@
   "devDependencies": {
     "@actions/core": "^1.10.1",
     "@actions/github": "^6.0.0",
-    "@babel/core": "^7.24.3",
+    "@babel/core": "^7.24.4",
     "@babel/eslint-parser": "^7.24.1",
     "@babel/plugin-transform-runtime": "^7.24.3",
-    "@babel/preset-env": "^7.24.3",
+    "@babel/preset-env": "^7.24.4",
     "@commitlint/cli": "^19.2.1",
     "@commitlint/config-conventional": "^19.1.0",
     "babel-jest": "^29.7.0",
     "babel-loader": "^9.1.3",
     "bundlewatch": "^0.3.3",
     "copy-webpack-plugin": "^12.0.2",
-    "css-loader": "^6.10.0",
+    "css-loader": "^6.11.0",
```

Comment:
why bump the packages now this seems risky no? especially that Shay had progressed a lot with the regression cycle.

Reply:
This is just a minor bump, to release v2 with latest dependencies.
I confirmed CSS files are being loaded after this.